CMakeLists.txt

```diff
@@ -1124,7 +1124,7 @@ configure_file(${PCRE2_CONFIG_IN} ${PCRE2_CONFIG_OUT} @ONLY)
 set(PCRE2_CONFIG_VERSION_IN  ${CMAKE_CURRENT_SOURCE_DIR}/cmake/pcre2-config-version.cmake.in)
 set(PCRE2_CONFIG_VERSION_OUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/pcre2-config-version.cmake)
 configure_file(${PCRE2_CONFIG_VERSION_IN} ${PCRE2_CONFIG_VERSION_OUT} @ONLY)
-install(FILES ${PCRE2_CONFIG_OUT} ${PCRE2_CONFIG_VERSION_OUT} DESTINATION cmake)
+install(FILES ${PCRE2_CONFIG_OUT} ${PCRE2_CONFIG_VERSION_OUT} DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/pcre2)
```

Comment:
there used to be  time when CMAKE_INSTALL_LIBDIR was lib64 for 64bit builds, and I pressume avoiding that variance made using a "bitness" agnostic directory.

I also wonder how that would work in Windows, where the path is usually just "cmake" (usually with some funny case ifgnoring but preserving variant).

could we check with the rest of the "cmake" communitiy around #115 for advice before merging this backward incompatible change?

Reply:
https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure suggests that this is viable on both Windows and Unix

Reply:
agree it is "viable", the pont I was making was how to minimize "backward compatibility" issues

ex: currently it "might" fail to allow locatiing the cmake files if ${CMAKE_INSTALL_LIBDIR} is (arguibly incorrectly) set to lib64 and the system is Windows.  Making this change ONLY if not Windows would "solve" that and IMHO was simple enough to do.

Reply:
I'm not sure what you really want as several have suggested the same thing by now
https://github.com/PCRE2Project/pcre2/pull/260/files#r1225428249

Reply:
funny enough you linked to my "ask", which was "what can we ut in the release notes to make sure whoever might be affected by this change fixes their setup".

I don't maintan any software that uses pcre2 and that would be using this code, so I am not the best person to provide that.

Indeed, I dont't even know how to create an application that could be affected, but assume one running Windows that uses cmake and linking with a PCRE2 that provides this file (meaning it was installed by running `cmake install` instead of the more common way to get PCRE2 installed which is calling `make install` as "root" while using a POSIX compatible shell, hence maybe mingw64 bash.

Reply:
Not sure if this answers your question but vcpkg does this too
https://github.com/microsoft/vcpkg/blob/master/ports/pcre2/portfile.cmake#L54

Reply:
some of the patches we are discosing were tested with vcpkg but had to be reverted before 10.34RC1 because they broke autotools bootstrapping.

it doesn't answer though how to document the hopefully small backward incompatibility that they introduce in their current form.